php/ext/google/protobuf/package.xml

```diff
@@ -10,8 +10,8 @@
   <email>protobuf-opensource@google.com</email>
   <active>yes</active>
  </lead>
- <date>2019-06-17</date>
- <time>09:34:50</time>
+ <date>2019-07-17</date>
```

Comment:
Why is this file changing, seems unrelated?  If the one on _master_ is stale, I'd suggest doing a separate PR to fix that.

Reply:
This file changed when I ran `update_version.py` to verify that the version in `Protobuf-cc.podspec` is updated automatically. I guess this change shouldn't checked in - maybe it should only be run when someone cuts cuts a release?
I'll revert this change.